docs/home/data/equipe.js

```diff
@@ -1,58 +1,30 @@
-const equipe = [
-  {
-    "nome": "Thayanne Luiza",
-    "url_github": "https://github.com/thayannevls"
-  },
-  {
-    "nome": "Fanny Vieira",
-    "url_github": "https://github.com/fanny"
-  },
-  {
-    "nome": "Héricles Emanuel",
-    "url_github": "https://github.com/hericlesme"
-  },
-  {
-    "nome": "José Renan",
-    "url_github": "https://github.com/JoseRenan"
-  },
-  {
-    "nome": "José Robson",
-    "url_github": "https://github.com/JRobsonJr"
-  },
-  {
-    "nome": "Juan Barros",
-    "url_github": "https://github.com/JuanBarros2"
-  },
-  {
-    "nome": "Julio Guedes",
-    "url_github": "https://github.com/juliobguedes"
-  },
-  {
-    "nome": "Lucas Medeiros",
-    "url_github": "https://github.com/lucasmedeiros"
-  },
-  {
-    "nome": "Matheus Alves",
-    "url_github": "https://github.com/alvesmatheus"
-  },
-  {
-    "nome": "Paulo Leitão",
-    "url_github": "https://github.com/paulojbleitao"
-  },
-  {
-    "nome": "Pedro Espíndula",
-    "url_github": "https://github.com/pedroespindula"
-  },
-  {
-    "nome": 'Ivyna Santino',
-    "url_github": 'https://github.com/ivynasantino',
-  },
-  {
-    "nome": "Jessé Souza",
-    "url_github": "https://github.com/jessescn"
-  },
-  {
-    "nome": "Rayla Medeiros",
-    "url_github": "https://github.com/RaylaMedeiros"
-  }
-]
\ No newline at end of file
+const githubApi = "https://api.github.com";
+
+gerarDadosEquipe = async () =>  {
+  const dados = await 
+  fetch(`${githubApi}/orgs/opendevufcg/public_members`)
+  .then(response => response.json())
+  .catch(error => console.log("Error Team Get" + error))
```

Comment:
É melhor usar `console.error`

```suggestion
  .catch(error => console.error("Erro ao recuperar erro: " + error))
```